JavaScript/Post_Tweets/index.js

```diff
@@ -0,0 +1,38 @@
+var Twit = require('twit')
+const fs = require('fs');
+var T = new Twit({
+  consumer_key:         'YOUR_CONSUMER_KEY',
+  consumer_secret:      'YOUR_CONSUMER_SECRET',
+  access_token:         'YOUR_ACCESS_TOKEN',
+  access_token_secret:  'YOUR_ACCESS_SECRET',
+})
+ 
+// post tweet 
+var text = "ENTER_TWEET_TEXT"
```

Comment:
While the User is running the Script, can you try invoking the User to input the Tweet Text. Hard-Coding stuff in Scripts doesn't fare well much.